WMFComponents/Sources/WMFComponents/Components/Article Tabs/WMFArticleTabsViewModel.swift

```diff
@@ -101,10 +109,18 @@ public class WMFArticleTabsViewModel: NSObject, ObservableObject {
     }
     
     func calculateImageHeight() -> Int {
-        // If text is scaled up for accessibility, use taller image for single column
         if UIApplication.shared.preferredContentSizeCategory.isAccessibilityCategory {
+            if UIDevice.current.userInterfaceIdiom == .pad {
+                return 425
+            }
             return 225
         }
+
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            let isLandscape = UIDevice.current.orientation.isLandscape
+            return isLandscape ? 160 : 110
```

Comment:
Using isLandscape here might cause an issue with split screen on iPad, maybe using horizontalSizeClass == .regular to check if it's landscape and full width. I think it's worth testing in this case.